docs/application/vstools/tools/emulator-manager.md

```diff
@@ -20,35 +20,35 @@ There are 2 different ways to access the Emulator Manager from Visual Studio:
 
 - In the Visual Studio menu, go to **Tools &gt; Tizen &gt; Tizen Emulator Manager**.
 
-   ![Launch the Emulator Manager in the Tools menu](media/em-vstoolbar1.png)
+    **Figure: Launch the Emulator Manager in the Tools menu**
 
-- On the Visual Studio toolbar, click **Launch Tizen Emulator**.
+    ![Launch the Emulator Manager in the Tools menu](media/em-vstoolbar1.png)
 
-   ![Launch the Emulator Manager in the Visual Studio](media/em-vstoolbar2.png)
+-   On the Visual Studio toolbar, click **Launch Tizen Emulator**.
 
 
```

Comment:
Did you removed image 'em-vstoolbar2.png' on purpose?

Reply:
Done